Add documentation for LIVISI integration (#23808)
home-assistant/home-assistant.github.io: source/_integrations/livisi.markdown

---
title: LIVISI SmartHome
description: Access and control your LIVISI Smart Home Controller (SHC) and its connected RWE/innogy devices.
ha_category:
  - Switch
ha_iot_class: Local Push
ha_release: 2022.12.0
ha_config_flow: true
ha_codeowners:
  - '@StefanIacobLivisi'
ha_domain: livisi
ha_platforms:
  - switch
ha_integration_type: integration

---
The LIVISI Smart Home integration allows you to connect your LIVISI Smart Home Controller (SHC) to Home Assistant. The SHC can control compatible devices from RWE/innogy connected to it.
 
The following devices are currently supported by this integration:
 
- RWE/innogy Smart Plug (PSS)
 
##Prequisites
 

> This integration is communicating with the local version of LIVISI SmartHome only. 

Reviewer: ```suggestion
This integration communicates with the local version of LIVISI SmartHome only. 
```

---
title: LIVISI SmartHome
description: Access and control your LIVISI Smart Home Controller (SHC) and its connected RWE/innogy devices.
ha_category:
  - Switch
ha_iot_class: Local Push

> featured: true

Reviewer 2: ```suggestion
```

> ha_release: 2022.8.5

Reviewer 2: ```suggestion
ha_release: 2022.12.0
```

ha_config_flow: true
ha_codeowners:
  - '@StefanIacobLivisi'
ha_domain: livisi
ha_platforms:
  - switch
ha_integration_type: integration

---
The LIVISI Smart Home integration allows you to connect your LIVISI Smart Home Controller (SHC) to Home Assistant. The SHC can control compatible devices from RWE/innogy connected to it.
 
The following devices are currently supported by this integration:
 
- RWE/innogy Smart Plug (PSS)
 
##Prequisites
 
This integration is communicating with the local version of LIVISI SmartHome only. 
 
To activate the local SmartHome functionality, please use the LIVISI App and go to **Settings** >> **General Settings** >> **Local SmartHome** and click **Activate**.
 
Please visit [LIVISI Community](https://community.livisi.de) for more information.
 
{% include integrations/config_flow.md %}
 
### Extra Configuration
 

> The current integration will not find you SHC automatically and needs to be configured manually. You will need to provide the IP address and the local password for the controller.

Reviewer 2: ```suggestion
The current integration will not find your SHC automatically and needs to be configured manually. You will need to provide the IP address and the local password for the controller.
```

 
## Device Discovery

> All devices are automatically discovered and included by the integration. If you include a new device in LIVISI SmartHome, it will automatically appear in Home Assistant after a few minutes.

Reviewer 2: ```suggestion
All devices are automatically discovered and included by the integration. If you include a new device in LIVISI SmartHome, the device will automatically appear in Home Assistant after a few minutes.
```

---
title: LIVISI SmartHome
description: Access and control your LIVISI Smart Home Controller (SHC) and its connected RWE/innogy devices.
ha_category:
  - Switch
ha_iot_class: Local Push
featured: true
ha_release: 2022.8.5
ha_config_flow: true
ha_codeowners:
  - '@StefanIacobLivisi'
ha_domain: livisi
ha_platforms:
  - switch
ha_integration_type: integration

---

> The `livisi` integration allows you to connect your LIVISI Smart Home Controller (SHC) to Home Assistant. The SHC can control compatible devices from RWE/innogy connected to it.

Reviewer: ```suggestion
The LIVISI SmartHome integration allows you to connect your LIVISI Smart Home Controller (SHC) to Home Assistant. The SHC can control compatible devices from RWE/innogy connected to it.
```

> The following devices  are currently supported by this integration:

Reviewer: ```suggestion
The following devices are currently supported by this integration:
```

 
- RWE/innogy Smart Plug (PSS)
 

> ##Preparing LIVISI SmartHome for Home Assistant

Reviewer: ```suggestion
## Prequisites
```

 
This integration is communicating with the local version of LIVISI SmartHome only. 
 
To activate the local SmartHome functionality, please use the LIVISI App and go to **Settings** >> **General Settings** >> **Local SmartHome** and click **Activate**.
 
Please visit [LIVISI Community](https://community.livisi.de) for more information.
 
{% include integrations/config_flow.md %}
 
### Extra Configuration
 
The current integration will not find you SHC automatically and needs to be configured manually. You will need to provide the IP address and the local password for the controller.
<div class="note">
This integration is communicating with the local version of LIVISI SmartHome only.

> </div>

Reviewer: I think we don't need a big note box for this. I guess just adding it to the prerequisites in the first paragraph in normal plain text, would work just as well.

> ## Device Discovery
> All devices are automatically discovered and included by the integration. If you include a new device in LIVISI SmartHome, it will automatically appear in Home Assistant after a few minutes.

Reviewer: ```suggestion
## Device Discovery

All devices are automatically discovered and included by the integration. If you include a new device in LIVISI SmartHome, it will automatically appear in Home Assistant after a few minutes.
```

### Device Names

> The device name used for the included device is the one used in LIVISI SmartHome. The names can be changed later through the Home Assistant interface via **Settings** >> **DEVICE NAME** .

Reviewer: This is describing default Home Assistant functionality, not specific to the integration. Lets remove it.

### Rooms

> The rooms of the devices are taken from LIVISI SmartHome. If a room does not exist in Home Assistant, the room will be automatically created.

Reviewer: In Home Assistant, these are called Areas. Nevertheless, this paragraph describes default Home Assistant behavior and functionality and could therefore be removed.
